k8s/conf/cluster-role-admin.yaml

```diff
@@ -16,3 +16,6 @@ rules:
   - apiGroups: ["apiextensions.k8s.io"]
     resources: ["customresourcedefinitions"]
     verbs: ["*"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get"]
```

Comment:
Interesting, why is that needed?

Reply:
It's needed to give nsmd-k8s ability to "get" configmap `kubeadm-config` from `kube-system` namespace. 
I think later we need more accurate role management and its binding.